client/me/purchases/manage-purchase/index.jsx

```diff
@@ -145,7 +184,7 @@ const ManagePurchase = React.createClass( {
 		const purchase = getPurchase( this.props ),
 			{ id, payment: { creditCard } } = purchase;
 
-		if ( isExpired( purchase ) || isOneTimePurchase( purchase ) || isIncludedWithPlan( purchase ) ) {
+		if ( isExpired( purchase ) || isOneTimePurchase( purchase ) || isIncludedWithPlan( purchase ) || ! getSelectedSite( this.props ) ) {
```

Comment:
Also I fail to see how `getSelectedSite( this.props )` is better than `this.props.selectedSite`. Sure it avoids some duplication of code but it hides the real name of the prop